sources/packages/backend/apps/workers/src/controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -56,8 +60,8 @@ export class ProgramInfoRequestController {
     await this.applicationService.updateProgramInfoStatus(
       job.variables.applicationId,
       job.customHeaders.programInfoStatus,
-      job.variables.selectedLocation,
-      job.variables.selectedProgram,
+      job.variables.studentDataSelectedLocation,
```

Comment:
If the student dynamic data is changing, then the student application form must also change right? I do not see the form definition in PR. Why so?

Reply:
There is no change in the student dynamic data. The renaming is only for the variables created on Camunda from the dynamic data.